src/pint/scripts/pintk.py

```diff
@@ -160,7 +158,7 @@ def main(argv=None):
     )
     args = parser.parse_args(argv)
 
-    log.setLevel("WARNING")  # only display warnings and higher in the terminal
+    logging.getLogger().setLevel("INFO")
```

Comment:
Why did this change from WARNING to INFO? @champagne-cmd set the default level to WARNING to reduce the unnecessary messages when running pink that cluttered the output

Reply:
It's kind of moot now that pintk has the level setting. But this was copying the level from the INFO in `plk.py`, which I think may be the one that took effect. But this could go back to WARNING if that's the right default.

Reply:
Actually, I don't think the INFO in plk.py did anything - the setLevel called within pintk took precedence, maybe because the setLevel wasn't included in the actual function called by pintk.py. I meant to delete that setLevel in plk.py.